doc/how_to_guides/stomp_planner/stomp_planner.rst

```diff
@@ -0,0 +1,109 @@
+      control_cost_weight: 0.1
+      delta_t: 0.1
+
+#. Configure MoveIt to load the STOMP planning pipeline by adding "stomp" to your MoveItConfiguration launch statement next to "ompl" and the other planners. You can find an example for this in the `demo.launch.py <https://github.com/ros-planning/moveit_resources/blob/ros2/panda_moveit_config/launch/demo.launch.py#L42>`_ of the Panda config.
+
+Running the Demo
+----------------
+If you have the ``panda_moveit_config`` from the `ros-planning/moveit_resources <https://github.com/ros-planning/moveit_resources>`_ repository you should be able to simply launch the demo setup and start planning with STOMP in RViZ ::
+
+  ros2 launch panda_moveit_config demo.launch.py
+
+STOMP Parameters
+----------------
+STOMP's parameters are configurable using the `stomp_planning.yaml <https://github.com/ros-planning/moveit_resources/blob/ros2/panda_moveit_config/config/stomp_planning.yaml>`_. All parameters are defined by the `stomp_moveit.yaml <https://github.com/ros-planning/moveit2/blob/main/moveit_planners/stomp/res/stomp_moveit.yaml>`_ which is used as input for the `generate_parameter_library <https://github.com/PickNikRobotics/generate_parameter_library>`_:
+
+**Optimization Parameters**:
+
+- *num_timesteps*: the number of timesteps used for trajectories - this number directly translates to the number of waypoints considered for trajectory initialization, during planning and for the solution output.
+
+- *num_iterations*: this is the total number of iterations that the planner can take to find a good solution while optimization.
+
+- *num_iterations_after_valid*: the number of iterations that the planner continues to optimize after finding an already valid solution.
+
+- *num_rollouts*: this is the number of noisy trajectories generated per iteration.
+
+- *max_rollouts*: the combined maximum number of old and new rollouts that are considered for optimization at any iteration.
+
+- *exponentiated_cost_sensitivity:* a factor for tweaking the exponentated costs used for probability calculation. Higher factors make STOMP optimize and converge quicker with the drawback of reduced robustness and less repeatable results.
+
+- *control_cost_weight*: this is the percentage of the trajectory accelerations cost to be applied in the total cost calculation.
+
+- *delta_t*: assumed time change between consecutive points.
+
+- *path_marker_topic*: Name of the topic RViZ subscribes to for path visualization.
+
+Choosing parameters for STOMP requires lesser intuition than CHOMP. One can have the default parameters for STOMP and this works well in most environments. However you could increase the number of timesteps or number of rollouts for STOMP to perform well under more complicated environments.
+
+
+Difference between plans obtained by STOMP, CHOMP and OMPL
+----------------------------------------------------------
+
+In this section a distinction is made between paths obtained from STOMP, CHOMP and OMPL. STOMP.
```

Comment:
```suggestion
In this section, a distinction is made between paths obtained from STOMP, CHOMP and OMPL.
```

Reply:
^ I added an extra comma to your suggestion too